api/docs/debug_memtrace.dox

```diff
@@ -108,9 +108,17 @@ Type cheat sheet (from trace_type_t enum):
 
 type_is_instr: 0xa-0x10 + 0x1e
 
+For .zip files, the data is split into the component files within the archive, in order.  Extract each component file in turn to view the data.  For example:
```

Comment:
Not sure I follow: do you mean the chunks aren't truncated due to missing data?  Doesn't it go without saying that things will break if data is missing?

Reply:
No. I mean a comment that says that each chunk will have the necessary repeated information (phys-virt mappings, timestamp/cpu markers, etc) so that the user doesn't have to go search the previous chunks (except some information in the very first chunk).

Reply:
Done.